### PR TITLE
feat(as-3732): enforcement notice dev and e2e

### PR DESCRIPTION
**Pull request by nssdesikan**
## Ticket Number
https://pins-ds.atlassian.net/browse/AS-3732

## Description of change
Development of Enforcement Notice page and its E2E tests

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by demzzydev — APPROVED** [review]
LGTM from a dev perspective

**Review by shilpabasu22 — APPROVED** [review]
LGTM

File: packages/e2e-tests/cypress/integration/full-planning/eligibility/enforcement-notice/enforcement-notice-steps.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import {
+  getEnforcementNoticeErrorMessage,
+  getEnforcementNoticeNo, getEnforcementNoticeYes,
+} from '../../../../support/full-planning/eligibility/page-objects/enforcement-notice-po';
+import { getContinueButton, getErrorMessageSummary } from '../../../../support/common-page-objects/common-po';
+import { verifyErrorMessage } from '../../../../support/common/verify-error-message';
+import { getBackLink } from '../../../../support/common-page-objects/common-po';
+const pageHeading = 'Have you received an enforcement notice?';
+const pageTitle = 'Have you received an enforcement notice? - Before you start - Appeal a planning decision - GOV.UK';
+const url = 'http://localhost:9003/before-you-start/enforcement-notice';
```
Comment (shilpabasu22) [inline packages/e2e-tests/cypress/integration/full-planning/eligibility/enforcement-notice/enforcement-notice-steps.js]: Why have we passed the complete url here?

**Review by shilpabasu22 — APPROVED** [review]
LGTM

**Pull request merged**